src/py/flwr/server/run_serverapp.py

```diff
@@ -136,32 +137,50 @@ def run_server_app() -> None:
             cert_path,
         )
 
-    log(
-        DEBUG,
-        "Flower will load ServerApp `%s`",
-        getattr(args, "server-app"),
+    if not (getattr(args, "server-app") is None) ^ (args.run_id is None):
+        raise sys.exit(
+            "Invalid options: Either `server-app` or `--run-id` "
+            "should be set but not both."
```

Comment:
```suggestion
            "Invalid options: Either `--server-app` or `--run-id` "
            "should be set but not both."
```

Reply:
Should we add `--` before the positional argument `server-app`? I am asking this because we will run the CLI like `flower-server-app server:app` or `flower-server-app --run-id 123`.

Reply:
Good question. I added it to make it clear that this is an argument, but it might also mislead users into thinking that it needs to be passed with `--server-app`. Just `server-app` looked a bit off. Is there a different way we could put this?